blueocean-config/src/main/java/io/jenkins/blueocean/config/BlueOceanConfig.java

```diff
@@ -55,14 +56,23 @@ private String createConfig() {
                             .key("allowAnonymousRead").value(allowAnonymousRead)
                             .endObject()
                         .key("enableJWT").value(BlueOceanConfigProperties.BLUEOCEAN_FEATURE_JWT_AUTHENTICATION)
```

Comment:
Don't we want to move enableJWT to the same feature set, inside properties file?